src/components/nav-logo/nav-logo.tsx

```diff
@@ -0,0 +1,96 @@
+import { Component, Element, EventEmitter, h, Host, Prop, Event } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-nav-logo",
+  styleUrl: "nav-logo.scss",
+  shadow: true
+})
+export class CalciteNavLogo {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+  @Element() el: HTMLCalciteNavLogoElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+  /** When `true`, visually highlight the component */
+  @Prop({ reflect: true }) active: boolean;
```

Comment:
Naming: should this be selected or is active correct terminoligy?

Reply:
I think `active` aligns with List, `selected` would align with Dropdown, Tab. No preference on this one.

Reply:
the reason behind `active` is the UX we are displaying to the user. for `selected` user expects the focus kind of border for the component where as for `active` it is just border-bottom.